contact.php

```diff
@@ -0,0 +1,108 @@
+<?php
+$name = $_POST['name'];
+$phone = $_POST['phone'];
+$email = $_POST['email'];
+$dropdown = $_POST['dropdown'];
+$message = $_POST['message'];
+ 
+$strValidationMessage = "";
+ 
+$to = "example@rawad.com";
+$subject = "New Message";
+ 
+if (isset($_POST['submit']))
+{
+  $name = $_POST['name'];
+  $phone = $_POST['phone'];
+  $email = $_POST['email'];
+  $message = $_POST['message'];
+ 
+  $boolValidateOK = 1;
+ 
+  if(trim($name)==="")
+  {
+    $boolValidateOK = 0;
+    $strValidationMessage .= "\nPlease Enter your name.<br />";
+  }
+ 
+ 
+    //Phone Number Valadation
+  $phone = str_replace("-","",$phone);
+  $phone = str_replace(".","",$phone);
+  $phone = str_replace("(","",$phone);
+  $phone = str_replace(")","",$phone);
+  $phone = str_replace(" ","",$phone);
+   
+ 
+  if(!is_numeric($phone)){//Checks if Phone Number are a value of Numbers.
+    $boolValidateOK = 0;
+    $strValidationMessage .= "The phone is not a number<br/>";  
+  }
+ 
+  //Valadating Phone Number
+  if(strlen($phone) != 10){
+    $boolValidateOK = 0;
+    $strValidationMessage .= "Please enter a proper 10-digit phone number<br/>";
+  }
+ 
+ //Function Of the Email Address
+  function validate_email( $senderemail )
```

Comment:
I like the idea of breaking modular logic out into functions. I typically try to put my functions at the bottom so as to not interrupt the program flow when someone is reading through it.